### PR TITLE
Zjs/issue 33

### PR DESCRIPTION
**Pull request by stayman**
#33 resolved

### CODE REVIEW

**Comment by FifthSurprise** [comment]
@stayman 

Can you check the method in games_controller for updating the game time zone?  I know it stores to database but I do not think it picks and updates the time zone correctly nor does it display in correct time zone?

**Comment by stayman** [comment]
It seems to be working fine for me

File: app/models/game.rb [hunk 1]
```diff
@@ -109,4 +109,8 @@ def export_data(round)
         "recurring_credits" => recurring_credits,
       }
   end
+
+  def next_round
```
Comment (FifthSurprise) [inline app/models/game.rb]: If we're going to override an attribute method like this, I suggest we put converting the time zone as a conditional.

Ex.

```ruby
def next_round(in_time_zone: nil)
     if in_time_zone
        super.in_time_zone(self.time_zone)
     else
        super
     end
end
```

File: db/seeds.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@
   control_message: "Welcome",
   activity: "All is quiet around the world.",
   alien_comm: false,
-  next_round: (Time.now() + 30*60),
+  next_round: (Time.now() + 30*60).utc,
```
Comment (FifthSurprise) [inline db/seeds.rb]: Not you but let's just set default time zone to Eastern.

File: app/assets/javascripts/angular/controllers/dashboard.js [hunk 3]
```diff
@@ -0,0 +1,172 @@
+angular.module('dashboardApp', ['timer', 'ngAnimate'])
```
Comment (FifthSurprise) [inline app/assets/javascripts/angular/controllers/dashboard.js]: I think your rebase adding something it should not have added...?  Can you please double-check?

File: app/models/game.rb [hunk 4]
```diff
@@ -109,4 +109,12 @@ def export_data(round)
         "recurring_credits" => recurring_credits,
       }
   end
+
+  def next_round(loalized=true)
```
Comment (FifthSurprise) [inline app/models/game.rb]: loalized ==> localized

**Pull request merged**